stream/excel_stream_test.go

```diff
@@ -0,0 +1,418 @@
+package excel_stream
```

Comment:
This package name seems wrong.  XLSX is an open format implemented by multiple pieces of software, not just Excel.  Given that it's already going to be prefixed with github.com/tealeg/xlsx why don't we just call this "stream" ?